interface/ceed-operator.c

```diff
@@ -791,6 +834,50 @@ int CeedOperatorGetFields(CeedOperator op, CeedInt *num_input_fields, CeedOperat
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Set the arbitrary points in each element for a CeedOperatorAtPoints.
+
+  Note: Calling this function asserts that setup is complete and sets the CeedOperator as immutable.
+
+  @param[in,out] op           CeedOperatorAtPoints
+  @param[in]     rstr_points  CeedElemRestriction for points in each element
```

Comment:
Looks good overall. I think we'll want a more explicit statement of what this restriction is.

Reply:
Agreed - I'll make this PR a priority tomorrow